### PR TITLE
Add additional fields to list view table

### LINKED ISSUE #121: Integrate other metrics into list view
Beyond just the placeholders form early development

Use the spreadsheet from the "PARAT metrics" spreadsheet - add all available metrics for now, then Zach can decide if any should be removed

### LINKED ISSUE #131: List view filter states "bounce" on refreshes
AKA "_party parrot filters_"

When one or more filters are applied in the list view and the pages is refreshed, the filter states "bounce" back and forth before eventually stabilizing.  Most of the recent times I've seen it the filter seems to stabilize on the initial/desired state, although early one I was seeing it stabilize on not having a filter value.

https://github.com/georgetown-cset/parat/assets/22353962/7740b2e5-84a9-4970-ac1c-26147b590693

Discovered in https://github.com/georgetown-cset/parat/pull/126#discussion_r1347631393

### PR DESCRIPTION
**Pull request by brianlove**
Add all currently-available data fields in the `articles`, `patents`, and `other_metrics` keys to the List View table.

Closes #121, closes #131

### CODE REVIEW

File: web/gui-v2/src/components/ListViewTable.jsx [hunk 1]
```diff
@@ -130,8 +130,38 @@ const DEFAULT_FILTER_VALUES = {
   country: [],
   continent: [],
   stage: [],
+
+  all_pubs: [0, 100],
```
Comment (jmelot) [inline web/gui-v2/src/components/ListViewTable.jsx:134]: Might be nicer to do this assignment in an iterator

File: web/gui-v2/src/components/ListViewTable.jsx [hunk 2]
```diff
@@ -242,10 +272,38 @@ const ListViewTable = ({
       continent: useQueryParamString('continent', initialVal('continent')),
       stage: useQueryParamString('stage', initialVal('stage')),
       // ...
+
+      all_pubs: useQueryParamString('all_pubs', initialVal('all_pubs')),
```
Comment (jmelot) [inline web/gui-v2/src/components/ListViewTable.jsx:276]: again I wonder if iteration could make sense here, it's a lot of boilerplate

File: web/gui-v2/src/static_data/table_columns.js [hunk 3]
```diff
@@ -33,38 +63,184 @@ export default [
   { title: "Country", key: "country", initialCol: true, type: 'dropdown' },
   { title: "Region", key: "continent", initialCol: true, type: 'dropdown' },
   { title: "Stage", key: "stage", initialCol: true, type: 'dropdown' },
+
+  {
```
Comment (jmelot) [inline web/gui-v2/src/static_data/table_columns.js:67]: More boilerplate!

**Review by jmelot — COMMENTED** [review]
Just some comments about boilerplate (it's less maintainable and readable, I think we should wrap this up in iterators unless there's a compelling reason not to I'm not thinking of), I don't see any more substantive issues here

**Comment by jmelot** [comment]
When I tried this out in the UI I noticed two issues:

1.) AI pubs column is super wide for some reason

Screenshot 2023-10-04 at 3 34 12 PM

2.) When I add all cols, scroll all the way to the right, and sort by AI jobs, things get weird

Screenshot 2023-10-04 at 3 35 07 PM

File: web/gui-v2/src/components/ListViewTable.jsx (hunk 2 above)
Comment (brianlove) [reply web/gui-v2/src/components/ListViewTable.jsx:276]: Well, looks like there's a problem after all - refreshing the page leads to the filter state bouncing back and forth:

https://github.com/georgetown-cset/parat/assets/22353962/4692be32-2cb8-4a59-8d7d-d2d3b424dca2

Comment (jmelot) [reply web/gui-v2/src/components/ListViewTable.jsx:276]: Yikes, good catch. Is this a pre-existing issue? If so maybe makes sense to open a separate issue for it

Comment (brianlove) [reply web/gui-v2/src/components/ListViewTable.jsx:276]: Tracked in #131

**Comment by brianlove** [comment]
RE the AI/TT1 jobs column weirdness: The main part of the weirdness looks to have been resolved by changing how `null` values are treated and displayed (previously they were sorted above all other values, now they're sorted at the other end with zero).  The remaining issues can be resolved by adjusting the minimum column width.

**Review by jmelot — COMMENTED** [review]
Looking much nicer! As discussed, one follow-on in https://github.com/georgetown-cset/parat/issues/134 . I do have some styling notes which I'll follow up with in the comments

**Comment by jmelot** [comment]
Could use more padding in the last column

Screenshot 2023-10-11 at 10 05 29 AM

Dropdowns too narrow here (this is with all columns selected because I'm a pathological "user")

Screenshot 2023-10-11 at 10 06 47 AM

**Comment by brianlove** [comment]
Addressed cell ranking spacing in this PR and tracking column dropdown width in #143.

**Pull request merged**